scss/navbar.scss

```diff
@@ -0,0 +1,26 @@
+.navbar {
+    overflow: hidden;
+    background-color: white;
+    border-bottom: 1px solid black;
+    position: fixed; 
+    top: 0; 
+    height: $nav-bar-height;
+    width: 100%;
+    z-index: 9999;
+}
+
+.navbar .logo {
+    padding: 30px 20px;
+    height: $nav-bar-height;
+    width: auto;
+}
+
+.navbar .right-title {
+    font-family: $font-stack;
+    font-size: $text-big;
+    float: right;
+    line-height: $text-big;
+    padding: 33px 5px;
```

Comment:
you don't need a padding here, you can solve the vertical position with align items in the navbar